platform/commonUI/general/src/directives/MCTPopup.js

```diff
@@ -51,6 +51,8 @@ define(
                     position = [ rect.left, rect.top ],
                     popup = popupService.display(div, position);
 
+                div.css('z-index', 75);
```

Comment:
Actually ignore above, just realised this is the workaround you were talking about.